src/GraphQL.Tests/Types/IdGraphTypeTests.cs

```diff
@@ -11,9 +10,9 @@ public class IdGraphTypeTests
         private readonly IdGraphType _type = new IdGraphType();
 
         [Fact]
-        public void parse_value_null_throws()
+        public void parse_value_null_returns_null()
```

Comment:
Any tests for accepting Guid, Long, etc. for ID scalar ?

Reply:
Added a bunch of tests.